modular_chomp/code/modules/mob/living/silicon/robot/sprites/syndicate.dm

```diff
@@ -0,0 +1,72 @@
+/datum/robot_sprite/dogborg/syndie/v2dogborgsyndie
+	name = "Dogborg V2"
+	sprite_icon = 'modular_chomp/icons/mob/v2dogborgs_ch/v2dogborg_sprite.dmi'
+	sprite_icon_state = "vale"
+	has_eye_light_sprites = TRUE
+	has_custom_open_sprites = TRUE
+	has_vore_belly_sprites = TRUE
+	rest_sprite_options = list("Default", "Sit")
+/datum/robot_sprite/dogborg/protector/v2dogborgprotector
+	name = "Dogborg V2"
+	sprite_icon = 'modular_chomp/icons/mob/v2dogborgs_ch/v2dogborg_sprite.dmi'
+	sprite_icon_state = "vale"
+	has_eye_light_sprites = TRUE
+	has_custom_open_sprites = TRUE
+	has_vore_belly_sprites = TRUE
+	rest_sprite_options = list("Default", "Sit")
+/datum/robot_sprite/dogborg/mechanist/v2dogborgmechanist
+	name = "Dogborg V2"
+	sprite_icon = 'modular_chomp/icons/mob/v2dogborgs_ch/v2dogborg_sprite.dmi'
+	sprite_icon_state = "vale"
+	has_eye_light_sprites = TRUE
+	has_custom_open_sprites = TRUE
+	has_vore_belly_sprites = TRUE
+	rest_sprite_options = list("Default", "Sit")
+/datum/robot_sprite/dogborg/combat_medic/v2dogborgcombatmedic
+	name = "Dogborg V2"
+	sprite_icon = 'modular_chomp/icons/mob/v2dogborgs_ch/v2dogborg_sprite.dmi'
+	sprite_icon_state = "vale"
+	has_eye_light_sprites = TRUE
+	has_custom_open_sprites = TRUE
+	has_vore_belly_sprites = TRUE
+	rest_sprite_options = list("Default", "Sit")
+
+
+/datum/robot_sprite/dogborg/tall/combat_medic/dullahancombatmedic
```

Comment:
Why are they all named Dullahan Combat Medic? Dullahan is enough.

Reply:
2 am coding, il edit / delete